proto/buf/alpha/registry/v1alpha1/veracity.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+enum Veracity {
+  VERACITY_UNSPECIFIED = 0;
+  VERACITY_BUF_OFFICIAL = 1;
```

Comment:
Values need docs as well. Also I might consider something other than `BUF_OFFICIAL` - perhaps just `OFFICIAL`?

Also should this be in the registry package? Are we going to want to use this in the module package, for example?

Reply:
updated docs and to `OFFICIAL`.
I don't see that we will use it in the module package or other packages, as I think the furthest place that we plan to use it is in repository level, and probably not going indicate these status in a module